### ISSUE TITLE
Allow for flexible nPMTs in the model functions

### ISSUE THREAD

**Issue by cfuselli** [acolijn/OptoSim#9]
For example, in the event displays and in the weighted average there are some functions that work only with a 4 quadrant system.

**Comment by acolijn** [comment]
Yes. I am aware. This is just in some of the notebooks - not in the classes. Anyhow, some of the methods can only work for a 2x2 detector, like plotting (Q_left-Q_right)/sum(Q). UNless you do something very clever of course.
